src/com/epam/javatrainings/pizza/RegularPizza.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.javatrainings.pizza;
+
+import java.util.List;
+
+public class RegularPizza extends Pizza {
```

Comment:
I would not extend Regular from Pizza, it is not Pizza type, it's just pizza base type, so Pizza would have a field pizzaBase which will represent it.